modules/claims_api/app/sidekiq/claims_api/va_notify_follow_up_job.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+require 'jwt'
+
+module ClaimsApi
+  class VANotifyFollowUpJob < ClaimsApi::ServiceBase
+    NON_RETRY_STATUSES = %w[delivered preferences-declined].freeze
+    RETRY_STATUSES = %w[sent technical-failure temporary-failure permanent-failure].freeze
+
+    def perform(notification_id)
+      status = notification_response_status
+
+      unless NON_RETRY_STATUSES.include?(status)
+        self.class.perform_in(60.minutes, notification_id)
+        ClaimsApi::Logger.log(
+          'va_follow_up_job',
+          detail: "Status for notification #{notification_id} was #{status}"
+        )
+      end
+    rescue => e
+      ClaimsApi::Logger.log(
+        'va_follow_up_job',
```

Comment:
same as above. wonder if we just constant this or define it here to reuse so we only write it once.

Reply:
@FonzMP Yes I have that update already on my local branch with the LOG_TAG in line with what most (or all?) the other sidekiq job use